src/api/docs/content/index.html

```diff
@@ -10,14 +10,7 @@
         <script type='text/javascript' src='external/rapidoc-min.js'></script>
         <script type="text/javascript" src="pi-hole.js"></script>
         <link href='index.css' rel='stylesheet'>
-        <link rel="apple-touch-icon" href="/admin/img/favicons/apple-touch-icon.png" sizes="180x180">
-        <link rel="icon" href="/admin/img/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
-        <link rel="icon" href="/admin/img/favicons/favicon-16x16.png" sizes="16x16" type="image/png">
-        <link rel="manifest" href="/admin/img/favicons/manifest.json">
-        <link rel="mask-icon" href="/admin/img/favicons/safari-pinned-tab.svg" color="#367fa9">
-        <link rel="shortcut icon" href="/admin/img/favicons/favicon.ico">
-        <meta name="msapplication-TileColor" content="#367fa9">
-        <meta name="msapplication-TileImage" content="/admin/img/favicons/mstile-150x150.png">
+        <link rel="shortcut icon" href="img/favicon.ico">
```

Comment:
You are using the wrong directory:

```suggestion
        <link rel="shortcut icon" href="images/favicon.ico">
```